Lib/test/libregrtest/refleak.py

```diff
@@ -253,6 +253,7 @@ def dash_R_cleanup(fs, ps, pic, zdc, abcs, linecache_data):
         zipimport._zip_directory_cache.update(zdc)
 
     # Clear ABC registries, restoring previously saved ABC registries.
+    # ignore deprecation warning for collections.abc.ByteString
```

Comment:
what's this comment talking about?

Reply:
It was removed in https://github.com/python/cpython/commit/2f4db5a04d6fa7ba5c1c6b82b482dd7ca48f3382 so `git revert` added it back... I'll remove it again. I don't know what it's talking about 😄